OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -425,6 +436,10 @@ void StartSkirmishGame()
 			Game.Settings.Server.Map = map;
 			Game.Settings.Save();
 
+			var discordService = Game.ModData.Manifest.Get<DiscordService>();
+			if (discordService != null)
+				discordService.SetStatus("In Skirmish", DateTime.UtcNow);
```

Comment:
This is overwritten in the next tick by "In Lobby". Probably best to remove this from here, and change LobbyLogic to report "In Skirmish Lobby" or "In Multiplayer Lobby".

Campaign missions are currently not reported, with the status staying as "In Menu".